docs/content/preview/integrations/mirantis.md

```diff
@@ -0,0 +1,225 @@
+---
+title: Mirantis Kubernetes Engine
+linkTitle: Mirantis
+description: Mirantis
+menu:
+  preview_integrations:
+    identifier: mirantis
+    parent: integrations
+    weight: 571
+type: docs
+---
+
+[Mirantis Kubernetes Engine (MKE)](https://docs.mirantis.com/mke/3.5/overview.html) is a container orchestration platform for developing and running modern applications at scale, on private clouds, public clouds, and on bare metal.
+
+MKE as a container orchestration platform is specially beneficial in the following scenarios:
+
+- Orchestrating more than one container
+- Robust and scalable applications deployment
+- Multi-tenant software offerings
+
+The following sections describe how to deploy a single node YugabyteDB cluster on Mirantis MKE using [kubectl](https://kubernetes.io/docs/reference/kubectl/) and [helm](https://helm.sh/).
+
+This page describes the steps for a single node cluster for the purpose of simplicity, as you require more than one machine/VM for a multi node cluster deployment.
+
+## Prerequisite
+
+Before installing a single node YugabyteDB cluster using Mirantis, ensure that you have the Docker runtime installed on your localhost. To download and install Docker, select one of the following environments:
```

Comment:
Rephrase `on your localhost.` to something like `on the host on which you are installing MKE.`